opencrvs/opencrvs-core
# OCRVS-1992: Changes on section: Where did the death occur?

Ticket: https://jembiprojects.jira.com/secure/RapidBoard.jspa?rapidView=90&modal=detail&selectedIssue=OCRVS-1992&assignee=557058%3A6b822c32-61ee-4cfd-bfdd-e48295940a39

How To test: 
run the application as ZMB, 
start death event & 
observe the "Where did the death occur?" section 

FYI: The 'Residential address of deceased' is same as the Permanent address of earlier implementation.
